tests/Unit/Domain/Test_TagsTimeDependent.cpp

```diff
@@ -128,11 +119,10 @@ void test() noexcept {
       domain::Tags::InertialMeshVelocityCompute<Dim>>;
 
   MAKE_GENERATOR(gen);
-  const std::array<double, 3> velocity{{1.2, 0.2, -8.9}};
+  const DataVector velocity{Dim, 1.2};
```

Comment:
Why the change in value here?

Reply:
Below, on old lines 152-166, each component of the `std::array` was used to construct a `DataVector` of length 1 for each function of time (3 in total). Since we only need 1 function of time now, I just made the original `std::array` a `DataVector` of appropriate length (depending on the Dim).

Same for the other comments.

Reply:
But why the change from {1.2, 0.2, -8.9} to {1.2, 1.2, 1.2}?

Reply:
Oh the values are arbitrary so instead of adding `if` statements to construct a separate `DataVector` for each Dim I just did this since it was simpler.